### PR TITLE
Add promptUserNumber to showcase

### PR DESCRIPTION
**Pull request by nmattia**
It was missing.

### CODE REVIEW

File: src/frontend/src/showcase.ts [hunk 1]
```diff
@@ -144,8 +143,7 @@ const iiPages: Record<string, () => void> = {
     ),
   deviceSettings: () =>
     deviceSettings(userNumber, dummyConnection, simpleDevice, false),
-  loader: () => withLoader(() => new Promise(() => renderConstructing())),
```
Comment (frederikrothenberger) [inline src/frontend/src/showcase.ts:147]: Why are those removed?

Comment (nmattia) [reply src/frontend/src/showcase.ts:147]: Too many branches. Thanks!

**Review by frederikrothenberger — APPROVED** [review]
LGTM, thanks!
